profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -205,6 +205,15 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
     $settings = [];
   }
 
+  // NEPT-62: Overwriting the 'delai' field properties.
+  if (isset($settings['delai'])) {
+    $settings['delai']['#title'] = t('Requested delivery date');
+    $settings['delai']['#date_year_range'] = '-0:+1';
```

Comment:
originally it's -3;+3 https://github.com/ec-europa/platform-dev/blob/a8b9d9625be4d6109a05cb03430edad89e70d37e/profiles/common/modules/custom/tmgmt_poetry/plugins/tmgmt_poetry.ui.inc#L310 why you change it here? the ticket is about making it required only, isn't it?